src/components/Navbar/index.tsx

```diff
@@ -47,13 +47,13 @@ function Navbar() {
   return (
     <div className={styles.NavbarLayout}>
       <div className={styles.NavbarLeft}>
-        <ConunLogo className={styles.ConunLogo} />
         <Link href="/">
           <a>
-            <div className={styles.NetworkCircle}></div>
-            <div>Main net is live</div>
+            <ConunLogo className={styles.ConunLogo} />
           </a>
         </Link>
+        <div className={styles.NetworkCircle}></div>
+        <div>Main net is live</div>
```

Comment:
This should say mainnet with no space, something to change in next PR perhaps lol

Reply:
Actually, that's what Aziz kept telling me. haha! I will correct it